.github/workflows/integrations.yml

```diff
@@ -110,12 +110,15 @@ jobs:
       fail-fast: false
       matrix:
         config:
-        - os: windows-2019
+        - os: windows-2022
           site-packages: Lib/site-packages
-        - os: ubuntu-20.04
-          site-packages: lib/python3.9/site-packages
+        # TODO(DF): Disabled pending upstream changes to use system
+        # default libstdc++ - i.e. dropping CY22 support in favour of
+        # CY24.
```

Comment:
Why is this a TODO?

Reply:
The TODO is to un-comment those lines, which is done in https://github.com/OpenAssetIO/OpenAssetIO/pull/1392 (which can't be merged until downstream projects are updated).  I guess there was no real reason to add the TODO tag, it just helps draw attention in case it gets forgotten.

Reply:
OK, I'll check that the commenting out is undone in the next PR then :)